src/cf-buttons/src/atoms/buttons-with-icons.less

```diff
@@ -1,1549 +1,793 @@
 @import '../atoms/buttons.less';
 
-//
-// Button icons
-//
-
 // Icon directions
 
 .a-btn_icon-position-left {
   &:before {
     padding-right: unit( 10.5px / @btn-font-size, em );
-    border-right: 1px solid @btn-text; // Fallback
-    border-right: 1px solid fade( @btn-text, 40% );
+    border-right: 1px solid mix( @btn-bg, @btn-text, 50% );;
```

Comment:
Minor performance gains in using mix when the background is a solid, known color